lib/src/mui_suggestors/components/shared_props_migrators/hit_area.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:over_react_codemod/src/util/component_usage_migrator.dart';
+import 'package:over_react_codemod/src/util/component_usage.dart';
+import 'package:over_react_codemod/src/util.dart';
+
+mixin HitAreaPropMigrators on ComponentUsageMigrator {
+  /// Returns a string that can be used to add the props from the right-hand
```

Comment:
All of the logic in this file are pulled from the `MuiButtonMigrator` and is completely unchanged

Reply:
Is it not possible to share them across migrators?